File: hw/a25/question4.cpp
// Accept String Display only Digits

#include<iostream>
using namespace std;

void DisplayDigits(char *str)
{
    while(*str != '\0')
    {
        if(*str >= '0' && *str <= '9')
        {
            cout << *str;
        }

        str++;
    
    }
    cout << endl;
}

int main()
{
    char Arr[20];

    cout << "Enter String" << endl;
    cin.getline(Arr,20);

    DisplayDigits(Arr);

    return 0;
}

File: hw/a45/question4.java
// Addition of elements from each Column

import Marvellous.Matrix;
import java.lang.*;
import java.util.*;

class MyMatrix extends Matrix
{
    public MyMatrix(int a,int b)
    {
        super(a,b);        
    }
    
    public void AddColumn()
    {
        int iSum = 0;
        for(int i = 0; i < 1; i++)
        {
            for(int j = 0; j < iCol; j++)
            {
                iSum = iSum + Arr[i][j];

            }
        }
    }
}

class question1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Number of Rows");
        int iRow = sobj.nextInt();
        System.out.println("Enter Number of Columns");
        int iCol = sobj.nextInt();

        MyMatrix mobj = new MyMatrix(iRow,iCol);
        mobj.Accept();
        mobj.Display();
        mobj.AddColumn();
    }
}